input/script_utils.go

```diff
@@ -2117,34 +2117,46 @@ func (c *CommitScriptTree) CtrlBlockForPath(path ScriptPath,
 	}
 }
 
-// NewLocalCommitScriptTree returns a new CommitScript tree that can be used to
-// create and spend the commitment output for the local party.
-func NewLocalCommitScriptTree(csvTimeout uint32,
-	selfKey, revokeKey *btcec.PublicKey) (*CommitScriptTree, error) {
+func NewLocalCommitDelayScript(csvTimeout uint32,
+	selfKey *btcec.PublicKey) ([]byte, error) {
 
-	// First, we'll need to construct the tapLeaf that'll be our delay CSV
-	// clause.
 	builder := txscript.NewScriptBuilder()
 	builder.AddData(schnorr.SerializePubKey(selfKey))
 	builder.AddOp(txscript.OP_CHECKSIG)
 	builder.AddInt64(int64(csvTimeout))
 	builder.AddOp(txscript.OP_CHECKSEQUENCEVERIFY)
 	builder.AddOp(txscript.OP_DROP)
 
-	delayScript, err := builder.Script()
-	if err != nil {
-		return nil, err
-	}
+	return builder.Script()
+}
 
-	// Next, we'll need to construct the revocation path, which is just a
-	// simple checksig script.
-	builder = txscript.NewScriptBuilder()
+func NewLocalCommitRevokeScript(selfKey, revokeKey *btcec.PublicKey) ([]byte,
```

Comment:
Missing `godoc` comments here and above.

Reply:
Qualify in the funcotin names that both of these are for the taproot channels? They use schnorr keys, etc.